kernelci/test.py

```diff
@@ -108,7 +108,12 @@ def get_params(meta, target, plan_config, storage):
     rootfs = plan_config.rootfs
     defconfig_full = kernel['defconfig_full']
     defconfig = ''.join(defconfig_full.split('+')[:1])
-    endian = 'big' if 'BIG_ENDIAN' in defconfig_full else 'little'
+
+    if "endian" in kernel:
+        endian = kernel["endian"]
+    else:
```

Comment:
I don't think we need to keep backward compatibility, the code is expecting the data to be valid so it should always have the `endian` attribute.

Reply:
what happen if someone use kci_test against an old build ? I think we should keep compatibility for this case.